### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by TomAFrench**
The docs were quite out of date and the makefile didn't make a ton of sense and seemed broken.

I've made a couple of changes which should mean everyone can compile the docs on their own machine. I've also made a fresh start on the docs directory.

### CODE REVIEW

**Comment by TomAFrench** [comment]
The docs still need a good amount of love but this should make things easier.

**Comment by muyiwaolu** [comment]
@TomAFrench I agree. I wanted to completely look into overhauling documentation and updating the hosting.

File: docs/source/conf.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('/Users/muyiwa/Code/monzo-python'))
```
Comment (muyiwaolu) [inline docs/source/conf.py]: Jesus 😂

**Comment by adesnmi** [comment]
I'm going to scrap the current documentation approach, re-think this, and open another PR.

Related: ee0bf1958950c2d5f8edb05d1324577f0e99a303

**Pull request closed without merging**